codebases/openzaak.html

```diff
@@ -0,0 +1,116 @@
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <!--<a href="#img3" class="lb-btn lb-btn-prev"></a>-->
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img1"><img src="../photos/codebases/openzaak/openzaak_g01.png" alt="OpenZaak product example 1"></a>
+        <!--<a href="#img2" class="lb-btn lb-btn-next"></a>-->
+      </div>
+    </div>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We're helping launch the OpenZaak open source ecosystem by:</p>
+    <ul>
+      <li>Running a <a href="https://github.com/open-zaak/open-zaak-market-consultation/blob/develop/README.md" target="_blank">market consultation</a> to co-create codebase governance and community dynamics, and better understand vendor business models and risk sharing</li>
+      <li>Advising on decentralized procurement</li>
+      <li>Building the community through shared events, a mailing list and marketing materials</li>
+    </ul>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Technology</h3>
+    <dl>
+      <dt>Languages and frameworks:</dt>
+      <dd>Python</dd>
+    </dl>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Community</h3>
+    <dl>
+      <dt>Created by:</dt>
+      <dd>A coalition of more than 45 Dutch municipalities including Amsterdam, Arnhem, Delft, Haarlem, 's-Hertogenbosch, Rotterdam, Tilburg, Hoorn, Medemblik, Utrecht, SED (composed of Stede Broec, Enkhuizen and Drechterland) and <a href="https://www.dimpact.nl/" target="_blank">Dimpact</a> (composed of more than 30 municipalities) (2019)</dd>
+    </dl>
+    <dl>
+      <dt>Replicating:</dt>
+      <dd>The commissioning municipalities are currently looking for vendors who can offer them services using OpenZaak. They held a market consultation in spring 2020.</dd>
+    </dl>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Opportunity</h3>
+    <dl>
+      <dt>Scaling potential:</dt>
+      <dd>Other Dutch municipalities, other industries that use a case management system (like hospitals or hotels).</dd>
+    </dl>
+    <p>We're working closely with the commissioning municipalities to help them grow OpenZaak to a trusted codebase with:</p>
+    <ul>
+      <li>Many vendors offering a range of services using OpenZaak</li>
+      <li>Widespread adoption by municipalities</li>
+    </ul>
+    <p>OpenZaak maintainers see meeting the <a href="https://standard.publiccode.net" target="_blank">Standard for Public Code</a> as integral to achieving their vision.</p>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section cb-section-links">
+  <div class="cb-detail-info">
+    <h3>Find out more</h3>
+    <ul>
+      <li><a href="https://github.com/open-zaak/open-zaak" target="_blank">Github repo</a></li>
```

Comment:
I guess "repo" works, especially for tech-savvy people, but perhaps repository is easier English?